### PR TITLE
Fixed loading images from Flask server

### PR DESCRIPTION
**Pull request by MrMenezes**
Fixed loading images from Flask server.
Four types of images (.gif, .jpg, .tga, .png) images may be loaded. To load correctly, images must be located in the data directory of the current sketch.

### CODE REVIEW

File: pyp5js/http/web_app.py [hunk 1]
```diff
@@ -66,6 +68,12 @@ def render_sketch_view(sketch_name, static_path):
         elif not content_file.exists():
             return '', 404
 
+        if static_path.endswith(('gif', 'jpg', 'tga', 'png')):
```
Comment (berinhard) [inline pyp5js/http/web_app.py:71]: This tuple of file extensions could be refactored to a global constant so we can add a simple unit test to ensure we guarantee all the file formats supported by p5.js (only gif, jpg and png as said in [p5.Image docs](https://p5js.org/reference/#/p5.Image))

File: pyp5js/http/web_app.py [hunk 2]
```diff
@@ -66,6 +68,12 @@ def render_sketch_view(sketch_name, static_path):
...
+            img = open(content_file, 'rb').read()
+            return send_file(io.BytesIO(img),
+                             attachment_filename='%s.jpg' % static_path,
+                             mimetype='image/'+static_path[-3:])
```
Comment (berinhard) [inline pyp5js/http/web_app.py:75]: Can you add a unit test to ensure we're serving the file correctly?

File: pyp5js/http/web_app.py (hunk 1 above)
Comment (berinhard) [reply pyp5js/http/web_app.py:71]: Hm, I'm not sure why I've checked the `static_path` variable for extensions instead of the `content_file` path object, but we should definitely rely on the `content_file` because it's a `pathlib.Path` object and not a simple `str` one. My refactoring suggestion is:

```python
SUPPORTED_IMAGE_FILE_SUFFIXES = [".gif", ".jpg", ".png"]

.....

def render_sketch_view(....):
    # previous code here...

        if content_file.suffix in SUPPORTED_IMAGE_FILE_SUFFIXES:
             # serve img
```

We could also refactor the `if static_path.endswith('js')` condition to use the `Path.suffix` property as well:

```python
if content_file.suffix == '.js'
```

File: pyp5js/http/web_app.py (hunk 2 above)
Comment (berinhard) [reply pyp5js/http/web_app.py:75]: Edge case test scenario: static file names with upper case =]

If this test does not exist for the JS files, it would be nice to have it tested too.
